opentaskspal/src/main/java/org/dmfs/opentaskspal/rowsets/Subtasks.java

```diff
@@ -35,10 +36,11 @@
  */
 public final class Subtasks extends DelegatingRowSet<TaskContract.Tasks>
 {
-
-    public Subtasks(String authority, ContentProviderClient client, @NonNull RowSnapshot<TaskContract.Tasks> parentTask)
+    public Subtasks(@NonNull String authority, @NonNull ContentProviderClient client, @NonNull Uri parentTask, String... projection)
     {
-        super(new QueryRowSet<>(new TasksView(authority, client), new ReferringTo<>(TaskContract.Tasks.PARENT_ID, parentTask)));
+        super(new QueryRowSet<>(
+                new TasksView(authority, client, projection),
+                new EqArg(TaskContract.Tasks.PARENT_ID, ContentUris.parseId(parentTask))));
```

Comment:
What's wrong with the `ReferringTo` version? I'd definitely prefer it over the `EqArg` solution. To create a `RowReference` from a `Uri` use `RowUriReference`.

Reply:
I think I just didn't find that `RowUriReference`.. Updated now to use `ReferringTo`.